### PR TITLE
fix:ant-table unable to request(#2613)

### LINKED ISSUE #2613: chrome 浏览器无限loading
### Issue description

chrome 浏览器偶尔不停 loading , 单独看响应是正常的

火狐浏览器正常

### Expected behavior

不要不停loading 

### How to Reproduce

image
点了上游 点服务

### Screenshots

_No response_

### Environment

- apisix 2.15.0
- OS win10
- OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
- etcd 3.5.0
- apisix-dashboard version 2.13.0
- Browser version  chrome 105.0.5195.102 x64

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Huanghaiting**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Accoding to this idea ( https://github.com/ant-design/pro-components/issues/1921 ) ,  I upgraded its version.

**Related issues**

fix/resolve #2613

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by Baoyuantop** [comment]
Hi @Huanghaiting, can you help get the CI passed?
For failed frontend e2e tests, you can verify locally. Refer to https://apisix.apache.org/docs/dashboard/front-end-e2e/

**Comment by guoqqqi** [comment]
Hello, @Huanghaiting It looks like the test failed because some dom names changed due to the upgrade, you need to confirm and change the correct dom names:
image

File: web/cypress/e2e/rest/pluginTemplate-create-edit-delete-plugin-template.cy.js [hunk 1]
```diff
@@ -48,7 +48,9 @@ context('Create Configure and Delete PluginTemplate', () => {
     cy.visit('/');
     cy.contains('Route').click();
     cy.get(selector.empty).should('be.visible');
-    cy.contains('Advanced').should('be.visible').click();
+    cy.contains('Advanced').trigger('mouseover');
+    cy.wait(50);
```
Comment (guoqqqi) [inline web/cypress/e2e/rest/pluginTemplate-create-edit-delete-plugin-template.cy.js:52]: We should avoid abusing wait, and use some should assertions to ensure that the tests are stable.

File: web/cypress/e2e/rest/pluginTemplate-create-plugin-template-with-route.cy.js [hunk 2]
```diff
@@ -56,7 +56,9 @@ context('Create PluginTemplate Binding To Route', () => {
     cy.visit('/');
     cy.contains('Route').click();
     cy.get(selector.empty).should('be.visible');
-    cy.contains('Advanced').should('be.visible').click();
+    cy.contains('Advanced').trigger('mouseover');
+    cy.wait(50);
```
Comment (guoqqqi) [inline web/cypress/e2e/rest/pluginTemplate-create-plugin-template-with-route.cy.js:60]: ditto

File: web/cypress/e2e/rest/upstream-create_and_edit_upstream_with_no_nodes.cy.js [hunk 3]
```diff
@@ -61,9 +61,8 @@ context('Create and Delete Upstream', () => {
     cy.get(selector.nameSelector).type(data.upstreamName);
     cy.contains('Search').click();
     cy.contains(data.upstreamName).siblings().contains('Configure').click();
-
     cy.get(selector.upstreamNodeMinus0).should('not.exist');
-    cy.contains('button', 'Next').should('not.be.disabled').click();
+    cy.contains('button', 'Next').should('not.be.disabled').wait(10).click();
```
Comment (guoqqqi) [inline web/cypress/e2e/rest/upstream-create_and_edit_upstream_with_no_nodes.cy.js:65]: ditto

**Comment by Baoyuantop** [comment]
Hi @Huanghaiting, the frontend E2E tests are not very stable, so if there is an error, but it is verified locally by you, please let us know to help you re-run it.

File: web/cypress/e2e/rest/pluginTemplate-create-edit-delete-plugin-template.cy.js [hunk 4]
```diff
@@ -48,7 +48,12 @@ context('Create Configure and Delete PluginTemplate', () => {
     cy.visit('/');
     cy.contains('Route').click();
     cy.get(selector.empty).should('be.visible');
-    cy.contains('Advanced').should('be.visible').click();
+    cy.contains('Advanced')
+      .should('be.visible')
+      .click()
+      .then(() => {
+        cy.contains('Advanced').trigger('mouseover');
```
Comment (Baoyuantop) [inline web/cypress/e2e/rest/pluginTemplate-create-edit-delete-plugin-template.cy.js:55]: Are these sections required?

**Comment by Huanghaiting** [comment]
The drop-down filter needs to be triggered here, how to trigger it to make it effective and better? Do you have a better suggestion?
Another question, I see that some places trigger two clicks, like this file: pluginTemplate-create-plugin-template-with-route.cy.js
cy.contains('Next').click();
cy.contains('Next').click();
I don't know what it means and how does it work?
&nbsp;Someone else answered a similar question, but didn't tell the reason






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/apisix-dashboard"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月21日(星期五) 下午2:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/apisix-dashboard] fix:ant-table unable to request(#2613) (PR #2641)





 
@Baoyuantop commented on this pull request.
 
 
In web/cypress/e2e/rest/pluginTemplate-create-edit-delete-plugin-template.cy.js:
 &gt; @@ -48,7 +48,12 @@ context('Create Configure and Delete PluginTemplate', () =&gt; {      cy.visit('/');      cy.contains('Route').click();      cy.get(selector.empty).should('be.visible'); -    cy.contains('Advanced').should('be.visible').click(); +    cy.contains('Advanced') +      .should('be.visible') +      .click() +      .then(() =&gt; { +        cy.contains('Advanced').trigger('mouseover');  
Are these sections required?

**Comment by Huanghaiting** [comment]
please re-run it







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/apisix-dashboard"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月21日(星期五) 上午9:48
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/apisix-dashboard] fix:ant-table unable to request(#2613) (PR #2641)





 
Hi @Huanghaiting, the frontend E2E tests are not very stable, so if there is an error, but it is verified locally by you, please let us know to help you re-run it.

**Comment by Huanghaiting** [comment]
> The drop-down filter needs to be triggered here, how to trigger it to make it effective and better? Do you have a better suggestion? Another question, I see that some places trigger two clicks, like this file: pluginTemplate-create-plugin-template-with-route.cy.js cy.contains('Next').click(); cy.contains('Next').click(); I don't know what it means and how does it work? &nbsp;Someone else answered a similar question, but didn't tell the reason
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "apache/apisix-dashboard" ***@***.***&gt;; 发送时间:&nbsp;2022年10月21日(星期五) 下午2:54 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [apache/apisix-dashboard] fix:ant-table unable to request(#2613) (PR #2641) @Baoyuantop commented on this pull request. In web/cypress/e2e/rest/pluginTemplate-create-edit-delete-plugin-template.cy.js: &gt; @@ -48,7 +48,12 @@ context('Create Configure and Delete PluginTemplate', () =&gt; { cy.visit('/'); cy.contains('Route').click(); cy.get(selector.empty).should('be.visible'); - cy.contains('Advanced').should('be.visible').click(); + cy.contains('Advanced') + .should('be.visible') + .click() + .then(() =&gt; { + cy.contains('Advanced').trigger('mouseover'); Are these sections required? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***&gt;

File: web/cypress/e2e/rest/pluginTemplate-create-edit-delete-plugin-template.cy.js (hunk 4 above)
Comment (Huanghaiting) [reply web/cypress/e2e/rest/pluginTemplate-create-edit-delete-plugin-template.cy.js]: The drop-down filter needs to be triggered here, how to trigger it to make it effective and better? Do you have a better suggestion?
Another question, I see that some places trigger two clicks, like this file: pluginTemplate-create-plugin-template-with-route.cy.js
cy.contains('Next').click();
cy.contains('Next').click();
I don't know what it means and how does it work?
Someone else answered a similar question, but didn't tell the reason
image

Comment (Huanghaiting) [reply web/cypress/e2e/rest/pluginTemplate-create-edit-delete-plugin-template.cy.js]: please re-run it
image
image

**Review by Baoyuantop — APPROVED** [review]

**Review by oil-oil — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Comment by bzp2010** [comment]
@Huanghaiting Thank you for your contribution.

**Pull request merged**